test/extensions/filters/http/ext_authz/ext_authz_test.cc

```diff
@@ -1244,8 +1244,7 @@ TEST_F(HttpFilterTestParam, OverrideEncodingHeaders) {
                          {Http::LowerCaseString{"bar"}, "foo"},
                          {Http::LowerCaseString{"set-cookie"}, "cookie1=value"},
                          {Http::LowerCaseString{"set-cookie"}, "cookie2=value"},
-                         {Http::LowerCaseString{"accept-encoding"}, "gzip"},
-                         {Http::LowerCaseString{"accept-encoding"}, "deflate"}};
+                         {Http::LowerCaseString{"accept-encoding"}, "gzip,deflate"}};
```

Comment:
note: this change is due to the fact that accept-encoding is a request header, so in this case when it's specified as a response header it's no longer inline. This is a well known deficiency of the current append logic which is already known. Let me know if you want me to address this part of the change in a different way.

Reply:
Accept-Encoding used as a response header in the protocol between the ext-authz filter and the auth server?

I think this is fine but may be worth calling out in comments.

Reply:
It's just a test artifact of using a header as a response header that isn't actually a response header.